sbws/core/stats.py

```diff
@@ -130,7 +130,7 @@ def print_stats(args, data):
         fastest_transfer/1024))
     print('Results come from', first, 'to', last, 'over a period of',
           duration)
-    if args.error_types:
+    if getattr(args, 'error_types', False) is True:
```

Comment:
I think we can avoid this change. See elsewhere for suggested change to _PseudoArguments.

Reply:
i changed that to ``if hasattr(args, 'error_types')`` before i pushed, but forgot to commit :)